templates/tfengine/recipes/audit.hcl

```diff
@@ -101,7 +101,7 @@ schema = {
         <https://cloud.google.com/logging/docs/view/query-library> for query syntax.
         Need to escape \ and " to preserve them in the final filter strings.
         See example usages under "examples/tfengine/".
-        Logs with filter "logName:\"logs/cloudaudit.googleapis.com\"" is always enabled.
+        Logs with filter `"logName:\"logs/cloudaudit.googleapis.com\""` is always enabled.
```

Comment:
why do we need to add ` here? If ` is added here, then the backspash might need to be removed. If you didn't test it with Cloud Logging, maybe keep it as before.

Reply:
The complete paragraph explains that it is an example of how to escape quotes `"`. Afaik it's intended to be escaped since that's the example.

Am I wrong?

Reply:
Update: You're right! I misunderstood. I'll update it with the description suggested in the other comment

Reply:
After looking at https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/blob/master/examples/tfengine/multi_envs.hcl#L194, i think both are wrong. We should follow the pattern there, so that the generated configs will be like https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/blob/master/examples/tfengine/generated/multi_envs/audit/main.tf#L77

Reply:
This particular line is the final result at `.tf` file. I think it's good as it is right now, since it's just specifying the output of the config. 

Agree that we should follow https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/blob/master/examples/tfengine/multi_envs.hcl#L194 while writing configs, but that's already specified at the field description.

I'm a little confused haha

Reply:
Resolved offline. The misunderstanding is from we are looking at different things. I was looking at the raw md file, and Ernesto was looking at the rich text. All good.